PWGHF/Tasks/taskMcEfficiency.cxx

```diff
@@ -177,12 +199,18 @@ struct HfTaskMcEfficiency {
         if (isHypoMass1TrackStep) {
           if (pdgCode == pdg::kLambdaCPlus) {
             massHypo1 = invMassLcToPKPi(candidate);
+          } else if (pdgCode == pdg::kDPlus) {
+            massHypo1 = invMassDplusToPiKPi(candidate);
+          } else if (pdgCode == pdg::kDS) {
+            massHypo1 = invMassDsToKKPi(candidate);
           }
           hCandidates->Fill(kHFStepTracked, pt, massHypo1, pdgCode, cpa, collisionMatched, origin);
         }
         if (isHypoMass2TrackStep) {
           if (pdgCode == pdg::kLambdaCPlus) {
             massHypo2 = invMassLcToPiKP(candidate);
+          } else if (pdgCode == pdg::kDS) {
+            massHypo1 = invMassDsToPiKK(candidate);
```

Comment:
@fgrosa shouldn't it be `massHypo2` here?

Reply:
It was a typo indeed, fixed, thanks!